src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6368,6 +6763,36 @@ namespace HVACVariableRefrigerantFlow {
         int CapSizingMethod(0); // capacity sizing methods (HeatingDesignCapacity, CapacityPerFloorArea, FractionOfAutosizedCoolingCapacity, and
                                 // FractionOfAutosizedHeatingCapacity )
 
+        DataSizing::ZoneEqSizingData *select_EqSizing(nullptr);
```

Comment:
Changes to sizing to allow sizing for zone/airloop/OA sys equipment (similar to UnitarySystem sizing).